frc_characterization/logger_analyzer/templates/configs/pwfconfig.py

```diff
@@ -0,0 +1,47 @@
+{
+    # CAN device IDs for motors
+    # If doing drive test, treat this as the left side of the drivetrain
+    "motorPorts": [],
+    # Only if you are doing drive (leave empty "[]" if not)
+    "rightMotorPorts": [],
+    # Class names of motor controllers used.
+    # Options:
+    # 'CANVenom'
+    # If doing drive test, treat this as the left side of the drivetrain
+    "controllerTypes": [],
+    # Only if you are doing drive (leave empty "[]" if not)
+    "rightControllerTypes": [],
+    # Set motors to inverted or not
+    # If doing drive test, treat this as the left side of the drivetrain
+    "motorsInverted": [],
+    # Only if you are doing drive (leave empty "[]" if not)
+    "rightMotorsInverted": [],
+    # Encoder edges-per-revolution (*NOT* cycles per revolution!)
+    # encoderEPR is ignored when using the built in Venom encoder
+    "encoderEPR": 4,
```

Comment:
Is this a fixed value?

Reply:
No, it's not normally used unless someone wanted to use a standalone encoder.   I set this to 4 because the conversion from encoder counts to revolutions was encoderEPR.  I thought setting this to 4 would make the conversion become 4/4 or 1 and perhaps make it more obvious that Venom is already returning motor position in revolutions.

Happy to change this if you have a preference.

Reply:
The unnecessary division by 4 has been fixed, so you should be able to just keep it at 1.